core/rio/api/src/main/java/org/eclipse/rdf4j/rio/helpers/RDFaVersion.java

```diff
@@ -15,7 +15,6 @@
  *
  * @author Peter Ansell
  */
-@Deprecated(since = "4.3.0", forRemoval = true)
```

Comment:
Could you add a comment explaining that these RDFa settings are in use by some users and referencing this issue: https://github.com/eclipse-rdf4j/rdf4j/issues/4779